examples/chart/teleport-cluster/charts/teleport-operator/operator-crds/resources.teleport.dev_botsv1.yaml

```diff
@@ -35,6 +35,12 @@ spec:
           spec:
             description: Bot resource definition v1 from Teleport
             properties:
+              max_session_ttl:
+                description: The max session TTL value for the bot's internal role.
+                  Unless specified, bots may not request a value beyond the default
+                  maximum TTL of 12 hours. This value may not be larger than 168 hours.
```

Comment:
nit: Is there a reason to say "168 hours" rather than "7 days". It just feels slightly less intuitive to me.

Reply:
Yeah - definitely a nit but I'll usually do something like `7 days (168 hours)` for longer time periods in customer facing documentation.

Reply:
The thought was to match units, but you're right, that flows a lot more naturally. I've updated the docstring!